configure.ac

```diff
@@ -41,12 +41,28 @@ dnl AM_INIT_AUTOMAKE.
 dnl
 AC_CHECK_PROG(MAKEINFO, makeinfo, makeinfo)
 
-define([revision], m4_esyscmd_s([git rev-list -1 --abbrev-commit HEAD]))
+m4_define([revision], m4_esyscmd_s([git rev-list -1 --abbrev-commit HEAD]))
+
+m4_define([version_from_env], m4_esyscmd_s([echo $EXPLICIT_VERSION]))
+m4_ifblank(version_from_env, [
+    m4_define([version_from_detect], m4_esyscmd_s([misc/determine-version.py]))
+    m4_if(m4_sysval, 0, [], [
+        m4_fatal([Could not determine CFEngine version. Please set EXPLICIT_VERSION in the environment or make sure all git tags are up to date.])
```

Comment:
Yes, I wasn't sure. My thinking is that we can see if it gets problematic and add a default later if we need to. Right now it would be nice to catch the cases where it fails.